src/components/Person.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Person({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age > 0 && (
+        <p className="Person__age">{`I am ${age}`}</p>
+      )}
+
+      {isMarried ? (
+        <p className="Person__partner">
+          {`My ${sex === 'm' ? `wife's` : `husband's`} name is ${partnerName}`}
+        </p>
+      ) : (
+        <p className="Person__partner">I am not married</p>
+      )}
```

Comment:
It's better to create separate variable before return statement where you can store the result of inner conditional statement. 
Second p is redundant here, do conditional rendering inside one p tag
```suggestion
      <p className="Person__partner">
         {isMarried
           ? {`My ${partner}'s name is ${partnerName}`}
           : ...
      </p>
```